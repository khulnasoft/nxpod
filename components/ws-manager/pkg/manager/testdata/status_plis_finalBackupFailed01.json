{
    "plis": {
        "metadata": {
            "name": "plis-foobaz",
            "namespace": "default",
            "selfLink": "/api/v1/namespaces/default/configMaps/ws-foobaz",
            "uid": "fabadddc-4351-11e9-aee4-080027861af1",
            "resourceVersion": "63952",
            "labels": {
                "gpwsman": "true",
                "headless": "false",
                "owner": "foobar",
                "metaID": "metameta",
                "workspaceID": "foobaz",
                "workspaceType": "regular"
            },
            "annotations": {
                "nxpod/failedBeforeStopping": "true",
                "nxpod/id": "foobaz",
                "nxpod/servicePrefix": "foobaz",
                "nxpod/url": "http://10.0.0.114:8082",
                "nxpod/plis": "{\"finalBackupComplete\": true,\"finalBackupFailure\":\"backup timed out\",\"lastPodStatus\":{\"id\":\"foobaz\",\"metadata\":{\"owner\":\"foobar\",\"metaId\":\"metameta\"},\"spec\":{\"workspaceImage\":\"nginx:latest\",\"url\":\"http://10.0.0.114:8082\"},\"phase\":5,\"conditions\":{\"failed\":\"container workspace ran with an error: exit code 137\"}}}",
                "prometheus.io/path": "/metrics",
                "prometheus.io/port": "23000",
                "prometheus.io/scrape": "true"
            }
        }
    }
}