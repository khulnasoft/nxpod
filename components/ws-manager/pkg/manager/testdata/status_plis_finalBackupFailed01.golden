{
    "status": {
        "id": "foobaz",
        "metadata": {
            "owner": "foobar"
        },
        "spec": {
            "url": "http://10.0.0.114:8082"
        },
        "phase": 6,
        "conditions": {
            "failed": "container workspace ran with an error: exit code 137; last backup failed: backup timed out. Please contact support if you need the workspace data.",
            "final_backup_complete": 1
        }
    }
}